locksmith/src/utils/middlewares/recaptchaMiddleware.ts

```diff
@@ -5,9 +5,11 @@ import config from '../../config/config'
 /**
  * A list of authenticated users who are making calls for which we should bypass the captcha veification
  */
-const allowList = ['0x61be315032235Ac365e39705c11c47fdaee698Ee'].map(
```

Comment:
This was my own adress for testing!

Reply:
Same question as before. Why not store it in the database? More flexibility.